programmers2/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,6 @@
+function solution(s) {
+    const arr = s.split("");
+    arr.sort();
+    arr.reverse();
+    return arr.join("");
```

Comment:
배열이 아니기에 arr이라는 변수선언은 잘못되었다.
아래와 같이 따로 선언하지않고 한줄에 입력가능하다.
```suggestion
   return s.split('').sort().reverse('').join('');
```